binancial/accountant.py

```
@@ -11,15 +11,17 @@
         
         self.id = 0
         
-        self.account = self._init_account(credit_usdt=start_usdt, debit_usdt=None)
+        self.account = self._init_account(credit_usdt=start_usdt,
+                                          debit_usdt=None)
         
         self.update_id()
         
-    def _init_account(self, credit_usdt):
+    def _init_account(self, credit_usdt, debit_usdt):
```

Ruff (0.8.2)

Missing return type annotation for private function `_init_account`

(ANN202)